src/openforms/registrations/contrib/zgw_apis/options.py

```diff
@@ -72,6 +72,14 @@ class ZaakOptionsSerializer(JsonSchemaSerializerMixin, serializers.Serializer):
         ),
         default="",
     )
+    product = serializers.CharField(
```

Comment:
Should be URLField, no?

```suggestion
    product = serializers.URLField(
```

Reply:
Ah yeah, thats a lot better. I've also renamed it to `product_url`